grails-forge-core/src/main/java/org/grails/forge/feature/test/GebWithTestcontainers.java

```diff
@@ -74,12 +74,7 @@ public boolean supports(ApplicationType applicationType) {
 
     @Override
     public String getDocumentation() {
-        return "https://github.com/grails3-plugins/geb#readme";
-    }
-
-    @Override
-    public String getThirdPartyDocumentation() {
-        return "https://www.gebish.org/manual/current/";
+        return "https://github.com/apache/groovy-geb";
```

Comment:
I think https://www.gebish.org/manual/current is the right link for third-party documentation.